### PR TITLE
Playground Improvements

### PR DESCRIPTION
**Pull request by anubra266**
- [x] Add responsive mode
CleanShot 2023-07-24 at 09 35 20@2x
CleanShot 2023-07-24 at 00 24 42@2x

- [x] Improve Errors
CleanShot 2023-07-24 at 11 40 24@2x
CleanShot 2023-07-24 at 11 41 41@2x

- [x] Fix segmented control checked state ui
- [x] Fix color mode switch size shift
- [x] Automatically copy URI after saving playground, and show toast feedback

### CODE REVIEW

File: playground/panda.config.ts [hunk 1]
```diff
@@ -39,12 +39,27 @@ export default defineConfig({
...
         },
       },
       recipes,
     },
   },
+  staticCss: {
+    css: [
+      {
+        properties: {
+          cursor: ['ew-resize', 'ns-resize', 'nwse-resize', 'nesw-resize'],
```
Comment (astahmer) [inline playground/panda.config.ts:58]: not sure this is needed since they are used in the `playground/src/components/Preview/responsive-border.ts` file

**Review by astahmer — APPROVED** [review]
insane work ! the responsive view is a really cool feature, well played

File: playground/panda.config.ts (hunk 1 above)
Comment (anubra266) [reply playground/panda.config.ts:58]: @astahmer It's for the `playground/src/hooks/useResponsiveView.ts` file.
When we're resizing, we change the body cursor. Show it shows that we're resizing.

CleanShot 2023-07-24 at 09 45 54@2x

Comment (astahmer) [reply playground/panda.config.ts:58]: Screenshot 2023-07-24 at 10 49 30
yes but they are used here so panda should be able to extract them already, no ?

Comment (anubra266) [reply playground/panda.config.ts:58]: You're right @astahmer

**Pull request merged**